meson_options.txt

```diff
@@ -5,6 +5,7 @@ option('enable-app', type: 'boolean', value: true)
 option('install-app', type: 'boolean', value: true)
 option('use_gym', type: 'boolean', value: false)
 option('enable-capi', type: 'boolean', value: true)
+option('enable-ccapi', type: 'boolean', value: true)
```

Comment:
did you test it if this enable-ccapi is false?

Reply:
I havent but I will.
This PR is getting a lot more work to be in a new commit with unittest/updated application to use cc api.

Reply:
> I havent but I will.
> This PR is getting a lot more work to be in a new commit with unittest/updated application to use cc api.

But this is not the adding new feature. It is related with build and I recommend to do the build if enable-ccapi is false. If it fails the people think main branch has bug.

Reply:
understood.